uk.co.compucorp.civicrm.tasksassignments/js/test/controllers/modal/modal-document_test.js

```diff
@@ -129,6 +145,31 @@ define([
       });
     });
 
+    describe('$scope.role', function () {
+      beforeEach(function () {
+        initController();
+      });
+
+      it('defaults to as admin', function () {
+        expect($scope.role).toBe('admin');
+      });
+    });
+
+    describe('isRole()', function () {
+      beforeEach(function () {
+        initController();
+        $scope.role = 'staff';
+      });
+
+      it('returns truthy value for correct username', function () {
```

Comment:
this test title doesn't make much sense related to what you are testing (what does the username has anything to do with it?)

```js
it('checks if the given role is the current one', function () {
  expect($scope.isRole('staff')).toBe(true);
  expect($scope.isRole('admin')).toBe(false);
});